tokenserver/views.py

```diff
@@ -96,6 +96,13 @@ def valid_assertion(request):
             raise _unauthorized("invalid-timestamp")
         raise _unauthorized("invalid-credentials")
 
+    try:
+        tokenVerified = assertion['idpClaims']['fxa-tokenVerified']
+    except KeyError:
+        tokenVerified = True
```

Comment:
Oh, good point...the `try/except` may indeed be cleaner overall than a double `.get` then